src/Bridge/Anthropic/ModelHandler.php

```diff
@@ -55,13 +61,28 @@ public function request(Model $model, object|array|string $input, array $options
         ]);
     }
 
+    /**
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     * @throws RedirectionExceptionInterface
+     * @throws DecodingExceptionInterface
+     * @throws ClientExceptionInterface
+     */
     public function convert(ResponseInterface $response, array $options = []): LlmResponse
     {
         if ($options['stream'] ?? false) {
             return new StreamResponse($this->convertStream($response));
         }
 
-        $data = $response->toArray();
+        $data = $response->toArray(false);
+
+        if (!isset($data['content']) || !count($data['content'])) {
```

Comment:
not a big fan of implicit casts, let's be explicit
```suggestion
        if (!isset($data['content']) || 0 === count($data['content'])) {
```